ras_party/clients/oauth_client.py

```diff
@@ -3,14 +3,13 @@
 
 class OauthClient:
     def __init__(self, config):
-        self.service_config = config.dependency['oauth2-service']
-        self.client_id = self.service_config['client_id']
-        self.client_secret = self.service_config['client_secret']
+        self.service = config['RAS_OAUTH_SERVICE']
```

Comment:
This is leftover from what we've previously been doing but I don't think it's good to be passing the config object around. Config should be set from`(current_)app.config[...]`